### PR TITLE
Randomization for business list

### PR DESCRIPTION
**Pull request by kaushiksrini**
A Fix for Issue #6 

Randomized the business lists instead of them showing in alphabetical order. 

Please do let me know if there needs to changes.

### CODE REVIEW

**Comment by kortina** [comment]
We actually had this in one version but decided to go back to alpha. If you want to make this a configurable setting and default to alpha sort, we could merge that in.

**Comment by kaushiksrini** [comment]
Sounds good! Changed to make the default config sort in alphabetical order.

File: index.html [hunk 1]
```diff
@@ -157,7 +158,19 @@
 
       sheet
         .filter(row => row.business_name != 'Business Name' && !!row.business_name)
-        .sort((a, b) => {return 0.5 - Math.random()})
```
Comment (kortina) [inline index.html]: Nice. Let's actually do something like the following.

```javascript
      const sortAlpha = (a, b) => {
        if (a.business_name.toLowerCase() < b.business_name.toLowerCase()) {
          return -1;
        }
        if (a.business_name.toLowerCase() > b.business_name.toLowerCase()) {
          return 1;
        }
        return 0;
      };
      const sortRandom = (a, b) => {
        return 0.5 - Math.random();
      };
      const sortFunc = RANDOMIZE_BUSINESS_LIST ? sortRandom : sortAlpha;

      sheet
        .filter(row => row.business_name != 'Business Name' && !!row.business_name)
        .sort(sortFunc)
        .map((row, i) => {
```

Comment (kaushiksrini) [reply index.html]: makes sense — made the appropriate changes

**Review by kortina — APPROVED** [review]
Cool, thanks!

**Pull request merged**